---a/net/minecraft/command/server/CommandSetDefaultSpawnpoint.java
+++b/net/minecraft/command/server/CommandSetDefaultSpawnpoint.java
@@ -1,6 +1,7 @@
 package net.minecraft.command.server;
 
 import java.util.List;
+
 import net.minecraft.command.CommandBase;
 import net.minecraft.command.CommandException;
 import net.minecraft.command.ICommandSender;
@@ -9,34 +9,29 @@
 import net.minecraft.server.MinecraftServer;
 import net.minecraft.util.BlockPos;
 
-public class CommandSetDefaultSpawnpoint extends CommandBase
-{
+public class CommandSetDefaultSpawnpoint extends CommandBase {
+
     private static final String __OBFID = "CL_00000973";
 
-    public String getName()
-    {
+    public String getName() {
         return "setworldspawn";
     }
 
-    public int getRequiredPermissionLevel()
-    {
+    public int getRequiredPermissionLevel() {
         return 2;
     }
 
-    public String getCommandUsage(ICommandSender sender)
-    {
+    public String getCommandUsage(ICommandSender sender) {
         return "commands.setworldspawn.usage";
     }
 
-    public void execute(ICommandSender sender, String[] args) throws CommandException
-    {
+    public void execute(ICommandSender sender, String[] args) throws CommandException {
         BlockPos blockpos;
 
-        if (args.length == 0)
-        {
+        if (args.length == 0) {
             blockpos = getCommandSenderAsPlayer(sender).getPosition();
-        }
-        else
-        {
-            if (args.length != 3 || sender.getEntityWorld() == null)
-            {
+        } else {
+            if (args.length != 3 || sender.getEntityWorld() == null) {
                 throw new WrongUsageException("commands.setworldspawn.usage", new Object[0]);
             }
 
@@ -48,11 +48,10 @@
 
         sender.getEntityWorld().setSpawnPoint(blockpos);
         MinecraftServer.getServer().getConfigurationManager().sendPacketToAllPlayers(new S05PacketSpawnPosition(blockpos));
-        notifyOperators(sender, this, "commands.setworldspawn.success", new Object[] {Integer.valueOf(blockpos.getX()), Integer.valueOf(blockpos.getY()), Integer.valueOf(blockpos.getZ())});
+        notifyOperators(sender, this, "commands.setworldspawn.success", new Object[]{Integer.valueOf(blockpos.getX()), Integer.valueOf(blockpos.getY()), Integer.valueOf(blockpos.getZ())});
     }
 
-    public List addTabCompletionOptions(ICommandSender sender, String[] args, BlockPos pos)
-    {
+    public List addTabCompletionOptions(ICommandSender sender, String[] args, BlockPos pos) {
         return args.length > 0 && args.length <= 3 ? func_175771_a(args, 0, pos) : null;
     }
-}
+}
